Description: this fixes an 'executable-not-elf-or-script' lintian warning.
Author: Tiago Ilieve <tiago.ilieve@gmail.com>

--- a/pythonpy/find_pycompletion.sh
+++ b/pythonpy/find_pycompletion.sh
@@ -1,3 +1,5 @@
+#!/bin/sh
+
 python -c '
 import os
 import pkg_resources
